src/main/java/com/github/manolo8/darkbot/gui/titlebar/ExtraButton.java

```diff
@@ -150,7 +152,13 @@ public void actionPerformed(ActionEvent e) {
                 ConfigEntity.changed();
                 main.getGui().updateConfiguration();
             }));
-
+            list.add(create("reset_stats", e -> {
+                main.statsManager.earnedUridium = 0;
+                main.statsManager.earnedCredits = 0;
+                main.statsManager.earnedExperience = 0;
+                main.statsManager.earnedHonor = 0;
+                main.guiManager.deaths = 0;
```

Comment:
This is force-setting values in statsManager, instead, it should be calling the method in stats manager that resets it

Keep in mind this is not resetting runtime for example, which is unintended behavior as it would break uri/h etc